### PR TITLE
Discover-upLoadImagesToPublicFolder

### PR DESCRIPTION
**Pull request by sundos-tamimi**
upLoad Images To The Public Folder

### CODE REVIEW

File: public/images/profilepic.png
Comment (duha-se) [inline public/images/profilepic.png:1]: Return all images that are moved to the public directory, to the assets file and use them from there.

File: src/pages/DiscoverPage/Discover.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from 'react';
+import { Box, InputBase, Grid } from '@mui/material';
+import VerticalTabs from '../../Components/VerticalTabs/VerticalTabs';
+import Appbar from '../../Components/AppBar/AppBar';
+import CardComponents from '../../Components/Card/Card';
+import { Search } from '@mui/icons-material';
+import post from '../HomePage/Posts';
+
+ {/* variable for test-Posts  */}
```
Comment (duha-se) [inline src/pages/DiscoverPage/Discover.jsx:9]: Delete all comments

File: src/pages/DiscoverPage/Discover.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useState } from 'react';
```
Comment (duha-se) [inline src/pages/DiscoverPage/Discover.jsx:1]: The order of import :
from react
mui /material
mui/icons
Components 
JSON
 Re-order them, please

File: src/pages/DiscoverPage/Discover.jsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+//       comments: [
+//         { username: 'Amer', comment: 'Nice dress' },
+//         { username: 'Dana', comment: 'I love it!' },
+//       ],
+//     },
+//     profilePic: '../public/images/profilepic.png',
+//   },
+// ];
+
+function Discover() {
```
Comment (duha-se) [inline src/pages/DiscoverPage/Discover.jsx:53]: use arrow function

File: src/pages/DiscoverPage/Discover.jsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+            alignItems: 'center',
+            borderRadius: '2rem',
+            padding: '1rem 2rem',
+            marginTop: '5vh',
+            width: '30vw',
+            bgcolor: '#e8f5e9',
+            gap: '1rem',
+          }}
+        >
+          <Search />
```
Comment (duha-se) [inline src/pages/DiscoverPage/Discover.jsx:123]: Change the bg color of the search input to #255b4e, and the icon search and the font color to #d0efce

File: src/pages/DiscoverPage/Discover.jsx [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+          <InputBase
+            placeholder="Search by location..."
+            value={searchTerm}
+            onChange={handleSearch}
+          />
+        </Box>
+          {/* Show posts */}
```
Comment (duha-se) [inline src/pages/DiscoverPage/Discover.jsx:130]: Screenshot 2024-10-18 at 12 35 11
I've noticed that the card's visual style could be improved to enhance its overall appearance and user experience. Additionally, I've encountered issues with the default location functionality. The user's current location is not being accurately detected or displayed.

**Review by duha-se — COMMENTED** [review]
Great Work , thanks

File: src/pages/DiscoverPage/Discover.jsx (hunk 2 above)
Comment (sundos-tamimi) [reply src/pages/DiscoverPage/Discover.jsx:1]: done

File: src/pages/DiscoverPage/Discover.jsx (hunk 3 above)
Comment (duha-se) [reply src/pages/DiscoverPage/Discover.jsx:53]: Rename to DiscoverPage

File: src/pages/DiscoverPage/Discover.jsx (hunk 4 above)
Comment (sundos-tamimi) [reply src/pages/DiscoverPage/Discover.jsx:123]: done

File: src/pages/DiscoverPage/Discover.jsx (hunk 1 above)
Comment (sundos-tamimi) [reply src/pages/DiscoverPage/Discover.jsx:9]: done

File: src/pages/DiscoverPage/Discover.jsx (hunk 3 above)
Comment (sundos-tamimi) [reply src/pages/DiscoverPage/Discover.jsx:53]: done

File: public/images/profilepic.png
Comment (sundos-tamimi) [reply public/images/profilepic.png:1]: done

**Pull request merged**